### PR TITLE
ISPN-9489 Adds missing compute family lifespan methods

### PR DESCRIPTION
**Pull request by karesti**
https://issues.jboss.org/browse/ISPN-9489

### CODE REVIEW

**Comment by karesti** [comment]
@danberindei I changed the java doc as you told in the other issue

File: client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/RemoteCacheSupport.java [hunk 1]
```diff
@@ -77,7 +77,9 @@ public final V replace(K key, V value, long lifespan, TimeUnit unit) {
    @Override
    public V put(K key, V value) {
       return put(key, value, defaultLifespan, MILLISECONDS, defaultMaxIdleTime, MILLISECONDS);
-   };
+   }
+
+   ;
```
Comment (tristantarrant) [inline client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/RemoteCacheSupport.java]: what did this semicolon do to you ? :)

Comment (tristantarrant) [reply client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/RemoteCacheSupport.java]: Remove it completely

Comment (karesti) [reply client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/RemoteCacheSupport.java]: ops

**Comment by karesti** [comment]
test failures not related

**Comment by karesti** [comment]
@danberindei done

**Comment by danberindei** [comment]
Thanks Katia, integrated!

**Pull request merged**